main/isotp_ble_bridge_main.c

```diff
@@ -90,15 +100,44 @@ static void twai_receive_task(void *arg)
     {
         twai_message_t rx_msg;
         twai_receive(&rx_msg, portMAX_DELAY); // If no message available, should block and yield.
-        if (rx_msg.identifier == receive_identifier)
-        {
-            ESP_LOGD(EXAMPLE_TAG, "Received Message with identifier %08X and length %08X", rx_msg.identifier, rx_msg.data_length_code);
-            for (int i = 0; i < rx_msg.data_length_code; i++)
-                ESP_LOGD(EXAMPLE_TAG, "RX Data: %02X", rx_msg.data[i]);
-            xSemaphoreTake(isotp_mutex, (TickType_t)100);
-            isotp_on_can_message(&isotp_link, rx_msg.data, rx_msg.data_length_code);
-            xSemaphoreGive(isotp_mutex);
-            xSemaphoreGive(isotp_wait_for_data);
+        ESP_LOGI(EXAMPLE_TAG, "Received TWAI message with identifier %08X and length %08X", rx_msg.identifier, rx_msg.data_length_code);
+        for (int i = 0; i < rx_msg.data_length_code; i++) {
+            ESP_LOGI(EXAMPLE_TAG, "RX Data: %02X", rx_msg.data[i]);
+        }
+        xSemaphoreTake(isotp_mutex, (TickType_t)100);
+        ESP_LOGI(EXAMPLE_TAG, "Took isotp_mutex");
+        switch (rx_msg.identifier) {
+            case 0x7E8: { // ECU
```

Comment:
if you can think of a better way to tackle this dynamically let me know. i was going for a sort of "hard coded" approach to J2534 message (and flow control) filters